config/pipeline.go

```diff
@@ -34,13 +34,26 @@ func LocalPipelineValues(parameters Parameters) Values {
 	vals := map[string]interface{}{
 		"pipeline.id":                                                "00000000-0000-0000-0000-000000000001",
 		"pipeline.number":                                            1,
+		"pipeline.name":                                              "",
 		"pipeline.project.git_url":                                   gitUrl,
 		"pipeline.project.type":                                      projectType,
 		"pipeline.git.tag":                                           git.Tag(),
 		"pipeline.git.branch":                                        git.Branch(),
 		"pipeline.git.revision":                                      revision,
 		"pipeline.git.base_revision":                                 revision,
 		"pipeline.git.branch.is_default":                             false,
+		"pipeline.git.commit.author_avatar_url":                      "",
+		"pipeline.git.commit.author_email":                           "",
+		"pipeline.git.commit.author_login":                           "",
+		"pipeline.git.commit.author_name":                            "",
+		"pipeline.git.commit.body":                                   "",
+		"pipeline.git.commit.subject":                                "",
```

Comment:
https://discuss.circleci.com/t/pipeline-git-commit-is-undocumented-useful-for-triggering-workflows-based-on-commit-message/52786

DO we need to update docs for this as well?

Reply:
Yeah, we've not done this yet as there's a bunch of new pipeline-values we're adding so it's worthwhile doing this all at the same time.

Reply:
@meeech I'm going toupdating docs for pipeline values next week!